platform/security/sts/security-sts-server/src/main/java/ddf/security/sts/SubjectDNConstraintsInterceptor.java

```diff
@@ -74,8 +75,8 @@ public void handleMessage(Message message) throws Fault {
 
     public Collection<Pattern> setSubjectDNPatterns(String subjectDNConstraints) {
```

Comment:
✏️  Per our coding standards, we should camel-case acronyms, so the class, variable, parameters and method names should `Dn` instead of `DN`. Not a big deal if it's too much trouble to rename all of those.